lib/routes/content.js

```diff
@@ -317,34 +304,31 @@ const routes = application => {
      * @param {object} res - HTTP Response
      * @param {object} next - Express callback
     */
-    app.post(`/${config.content.base}/:type/${config.content.actions.save}`, upload.any(), (req, res, next) => {
+    app.post(`/${config.content.base}/:type/${config.content.actions.save}`, (req, res, next) => {
       const referrer = _.get(req.session, 'referrer') || req.get('Referrer');
       let audits;
       let publishable = false;
       let check = 'publish';
       let files = [];
       let source = {
-        type: '',
+        type: 'add',
         id: '',
-        revision: '',
+        revision: Math.floor(1000000 + Math.random() * 9000000),
```

Comment:
`revision` content is a number, not a uuid. _Must_ be a number, (as opposed to blank, or string) or we get an integer error from knex.